src/elfpy/types.py

```diff
@@ -257,12 +263,16 @@ class RandomSimulationVariables:
 
     # dataclasses can have many attributes
     # pylint: disable=too-many-instance-attributes
-    target_liquidity: float = field(metadata="total size of the market pool (bonds + shares)")
-    target_pool_apr: float = field(metadata="desired fixed apr for as a decimal")
-    fee_percent: float = field(metadata="percent to charge for LPer fees")
-    vault_apr: list = field(metadata="yield bearing source APR")
-    init_vault_age: float = field(metadata="fraction of a year since the vault was opened")
-    init_share_price: float = field(default=None, metadata="initial market share price for the vault asset")
+    target_liquidity: float = field(metadata=to_description("total size of the market pool (bonds + shares)"))
+    target_pool_apr: float = field(metadata=to_description("desired fixed apr for as a decimal"))
```

Comment:
Might as well fix the rest of the `metadata` entries in this file, since you're creating `to_description`

Reply:
These are the only entries that break pyright, but sure I'll make them consistent.